### DISCUSSION TITLE
Need help on centering a button

### DISCUSSION THREAD

**Discussion by ahmnot** [beercss/beercss#213] [Q&A] [answered]
Hello,

I'm using beercss and it's great.

I have a little problem. I would like to horizontally center a button on a nav bar. Right now it's like this: 

![button Bar not centered](https://github.com/beercss/beercss/assets/16052099/00381db7-5c56-4704-8f0d-6d632120be68)

The button is the one with "Prendre rendez-vous" in it, in blue.

The html for this representation is this:

```
<header class="fill">
	<nav>
		<a class="circle transparent" href="/">
			<i class="primary-text" title="Accueil">favorite</i>
		</a>

		<div class="max" />
		<div class="max center-align">
			<a href="/rendez-vous">
				<button class="small-elevate primary-container">
					<i>event</i>
					<span class="large-text">Prendre rendez-vous</span>
				</button>
			</a>
		</div>
		<div class="max" />

		<a href="/rendez-vous">
			<button class="secondary-container">
				<i>account_circle</i>
				<span class="large-text">Connexion</span>
			</button>
		</a>
	</nav>
</header>

<main>
	<h1 class="center-align">Notre Doc</h1>
	<nav>
		<div class="max l" />
		<slot />
		<div class="max l" />
	</nav>
</main>
```

I can't seem to make it. Do you have any tips? Thank you very much in advance.

**Comment by leonardorafael** [comment, marked as answer]
Hi @ahmnot ,

This occurs because `max` try to get the same width on spaces.
If you want to center "in screen" you can use `absolute middle center` on element.
Try the codepen https://codepen.io/leo-bnu/pen/eYbEjMb

Is something like this:

![image](https://github.com/beercss/beercss/assets/1918010/0752f9e8-4d6e-46c1-8735-bc61b0836232)

Hope this helps you

**Reply by ahmnot** [reply]
Thanks a lot, I was missing the absolute class! Great
